types/src/trusted_state.rs

```diff
@@ -143,10 +141,10 @@ impl TrustedState {
         validator_change_proof: &'a ValidatorChangeProof,
     ) -> Result<TrustedStateChange<'a>> {
         let res_version = latest_li.ledger_info().version();
-        if res_version < self.latest_version {
-            // The response is stale so we don't update our trusted state.
-            return Ok(TrustedStateChange::Stale);
-        }
+        ensure!(
+            res_version >= self.latest_version,
```

Comment:
`res_version == self.latest_version` seems to be a valid case to me?

Reply:
right? that's what it's saying? : D

Reply:
ensure that the response version is greater-than-or-equal-to my current trusted version?